webcompat/templates/about.html

```diff
@@ -20,62 +20,62 @@ <h1 class="headline-1 highlight">About</h1>
         </p>
         <p>
-           You found something and are not sure if this is the right place to report that issue? Try to <strong class="bold">reproduce the issue</strong> in another browser. Does it work there? Try again in the first one. Still an issue? <strong class="bold">Congratulations!</strong> You found one. Send it in!
+           Have you found something and aren't sure if this is the right place to report that issue? Try to <strong class="bold">reproduce the issue</strong> in another browser. Does it work there? Try again in the first one. Still an issue? <strong class="bold">Congratulations!</strong> You found one. Send it in!
         </p>
         <p>
-          Having <strong class="bold">steps to reproduce</strong> in a bug report is very helpful for everyone who takes care of  triage.
+          Having <strong class="bold">steps to reproduce</strong> in a bug report is very helpful for everyone who takes care of triage and issue diagnosis.
         </p>
 
         <h4 class="headline-3">Bug Triage &amp; Diagnosis</h4>
         <p>
-          In the next step of the cycle, we take <strong class="bold">care of the reported bugs</strong>. One member of our community tries to reproduce the issue. When they can reproduce it, it moves to diagnosis. They might having an issue reproducing and will come back with questions. If the issue was already reported or is not a web compat issue, it will be closed. Don't forget, <strong class="bold">every web compat report we get is one we can take care of</strong>.
+          In the next step of the cycle, we take <strong class="bold">care of the reported bugs</strong>. One member of our community tries to reproduce the issue. When they can reproduce it, it moves to diagnosis. They might have an issue reproducing and will come back with questions. If the issue was already reported or is not a web compat issue, it will be closed. Don't forget, <strong class="bold">every web compat report we get is one we can take care of</strong>.
         </p>
         <p>
           After moving the issue to diagnosis, one member of our community digs into the code to figure out what’s causing the compatibility issue and or files bugs with browser vendors.
         </p>
 
         <p>
-          Sometimes it might also be possible that the official specification is just partially done or there is an issue in there. We help to <strong class="bold">provide feedback</strong> and make suggestions to the specifications to make it <strong class="bold">stable for everyone</strong>.
+          Sometimes it might also be possible the relevant web technology standard is underspecified or is poorly implemented. We help to <strong class="bold">provide feedback</strong> and make suggestions to the specifications to make them <strong class="bold">stable for everyone</strong>.
```

Comment:
I'm not sure about the word `poorly`. I know, it makes sense for a native speaker and fits, but might sound not appropriate for a non-native speaker. How about `not well` or something similar?

Reply:
Hmm. OK!